miden-lib/asm/miden/note.masm

```diff
@@ -181,8 +181,9 @@ end
 
 #! Computes hash of note inputs starting at the specified memory address.
 #!
-#! This procedure divides the hashing process into two parts: hashing pairs of words using
-#! `hash_memory_even` procedure and hashing the remaining values using the `hperm` instruction.
+#! This procedure divides the hashing process into two parts: hashing pairs of words using 
+#! `absorb_double_words_from_memory` procedure and hashing the remaining values using the `hperm`
+#! instruction. 
```

Comment:
Could we not replace most of this procedure with `rpo::hash_memory` procedure from the standard library?

Reply:
Ah, indeed! I wanted to remove this procedure after I ensure that everything is working, but forgot to do so yesterday. I'll replace the main body with `rpo::hash_memory` procedure.